src/main/resources/templates/privacy-policy.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" th:href="@{/webjars/govuk-frontend/4.5.0/govuk-frontend.min.css}"/>
+</head>
+
+<body class="govuk-template__body">
+<div th:replace="~{fragments/header :: header}"></div>
+
+<div class="govuk-width-container">
+    <a th:href="${homePageUrl}" class="govuk-back-link">Back</a>
+
+    <main class="govuk-main-wrapper" id="main-content" role="main">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <form method="post" th:action="@{/v2/privacy-policy}" th:object="${privacyPolicy}" novalidate>
+                    <div th:classappend="${#fields.hasErrors('acceptPrivacyPolicy') ? 'govuk-form-group--error' : ''}">
+                        <h1 class="govuk-heading-l">Do you accept our privacy policy?</h1>
+                        <p class="govuk-body">
+                            Before you can continue, you need to read and agree to the
+                            <a href="https://www.find-government-grants.service.gov.uk/info/privacy" target="_blank">
+                                Find a grant privacy policy (opens in a new tab).</a>
+                        </p>
+
+                        <div class="govuk-form-group">
+                            <p th:id="privacy-policy-error"
+                               th:class="govuk-error-message"
+                               data-cy="cy-email-input-validation-error-details"
+                               data-testid="error-message-test-id"
+                               th:errors="*{acceptPrivacyPolicy}"
+                            >
+                                <span className="govuk-visually-hidden">Error: </span>
```

Comment:
Can we render the error `message `here instead?

https://design-system.service.gov.uk/patterns/validation/#:~:text=If%20the%20user%E2%80%99s%20answers%20fail%20validation%3A

Reply:
Removed unneeded span, error message is rendered in the <p> tag as per design 😄

Reply:
thanks, looks good now :)